packages/wonder-blocks-pill/src/components/pill.tsx

```diff
@@ -4,17 +4,17 @@ import type {StyleDeclaration} from "aphrodite";
 
 import Clickable from "@khanacademy/wonder-blocks-clickable";
 import {View} from "@khanacademy/wonder-blocks-core";
-import {
-    Body,
-    LabelSmall,
-    LabelXSmall,
-} from "@khanacademy/wonder-blocks-typography";
+import {BodyText} from "@khanacademy/wonder-blocks-typography";
 import type {StyleType, AriaProps} from "@khanacademy/wonder-blocks-core";
 import type {ClickableRole} from "@khanacademy/wonder-blocks-clickable";
-import * as tokens from "@khanacademy/wonder-blocks-tokens";
+import {
+    semanticColor,
+    border,
+    color,
```

Comment:
I left a `NOTE` on the one remaining place we use the `color` tokens. It will get removed with the WB-1950 work!